src/types.ts

```diff
@@ -79,6 +79,8 @@ export interface PWAOptions {
    *
    * With this option enabled, the plugin will add the icons entry to your web manifest file.
    *
+   * If your web manifest file already contains the icons entry, the plugin will ignore this option.
```

Comment:
As i said, what if we changed the preset? this would not bring the new changes, which is wrong.

Reply:
The template has to be flexible enough, if the user decides not to use the default behavior, then it will be a bug if we change the icons

Reply:
https://github.com/QwikDev/pwa/pull/3#discussion_r1437004299 yes, it should not be a bug, it should change the entries, so I think we should remove the condition of checking if the icons exist or not. or at least watch the changes somehow.